kahuna/public/stylesheets/main.css

```diff
@@ -1192,7 +1192,12 @@ FIXME: what to do with touch devices
     border-bottom: 1px solid #565656;
 }
 
-.image-info:last-child .image-info__group{
+.image-info__group--last {
+    border-bottom: 0;
+    clear: both;
+}
+
+.image-info:last-child .image-info__group {
```

Comment:
That must have been left there from somewhere, I've removed it.
One was targeting `.image-info:last-child` were as I wanted the last `.image-info__group`.

I suppose I could use `.image-info__group:last-child` - but I try to avoid it to be explicit.